routes/map.js

```diff
@@ -1,116 +1,217 @@
 let express = require("express");
 const fs = require("fs");
 let router = express.Router();
+const { client } = require("./dbhandler")
 
 /**
- * Root map endpoint, get request, returns all names including the team they belong to
+ * Root map endpoint, expects a userID value and returns a list of objects that contain userName
+ * and mapID that the userID owns. 
  */
 
 router.get('/', (req, res, next) => {
-	res.send(JSON.parse(fs.readFileSync(__dirname + "/../public/tempDB.json", 'utf-8')));
+  // Expects userID
+  const { userID } =  req.body
+
+  // Ensure userID is specified
+  if (!userID && userID !== 0) {
+    res.status(400).send("Must specify userID!");
+  }
+  else {
+    // Connect to the database
+    client.connect(() => {
+      // Navigate to the correct database and collection
+      let database = client.db("blurp")
+      let collection = database.collection("maps")
+
+      // Find user by userID
+      collection.find({ userID: userID })
```

Comment:
As opposed to sending each entire map, maybe send just a list of the map IDs - we can extend that later if/when map names and other identifying data may be desired on the front end